src/controllers/ControllerPointer.js

```diff
@@ -262,7 +272,24 @@ FORGE.ControllerPointer.prototype._pinchMoveHandler = function(event)
 
     event.data.preventDefault();
 
-    this._camera.fov = this._pinchStartFov * event.data.scale;
+    var scale = event.data.scale;
+    var fovMin = this._camera.fovMin;
+    var fovMax = this._camera.fovMax;
+
+    var tmpFov = this._pinchStartFov * scale / this._pinchScaleFactorCorrection;
+
+    if (tmpFov < fovMin)
+    {
+        this._pinchScaleFactorCorrection = this._pinchStartFov * scale / fovMin;
+    }
+    else if (tmpFov > fovMax)
+    {
+        this._pinchScaleFactorCorrection = this._pinchStartFov * scale / fovMax;
+    }
+
+    tmpFov = this._pinchStartFov * scale / this._pinchScaleFactorCorrection;
```

Comment:
Wouldn't it be more intuitive to be the inverse ? Like `tmpFov = this._pinchStartFov / scale * this._pinchScaleFactorCorrection;`. It makes more sense in the movement of the user in my mind, but that's debatable.